samcli/lib/providers/provider.py

```diff
@@ -504,6 +504,7 @@ def cors_to_headers(cors: Optional["Cors"], request_origin: str) -> Dict[str, Un
 
         # cors.allow_origin can be either a single origin or comma separated list of origins
         allowed_origins = cors.allow_origin.split(",") if cors.allow_origin else list()
+        allowed_origins = [s.strip() for s in allowed_origins]
```

Comment:
nit: Can we rename `s` to avoid single character variable names (even though its just used in a loop that is being assigned back to itself)